src/CloudNative.CloudEvents/CloudNative.CloudEvents.csproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Memory" Version="4.5.5" Condition="'$(TargetFramework)'=='netstandard2.0' or '$(TargetFramework)'=='netstandard2.1'" />
+    <PackageReference Include="System.Memory" Version="4.5.5" Condition="'$(TargetFramework)'=='netstandard2.0'" />
```

Comment:
Might this one still be useful? The current version was released in May 2022, which is after .NET 6 was released... so it may contain bugfixes that are useful?

Reply:
No issue in re-adding it if you would like. Alternative suggestion, do we even need net-standard 2.1 (note I have not checked for framework conditional code).

Reply:
Fine for me. Don't see a need to spend too much time on the framework let me know the preference ie re-add for net standard 2.1 or leave it as proposed.

Reply:
Let's re-add netstandard2.1 - @captainsafia can always disagree :)

Reply:
Ok will re-add it tomorrow morning my time (ie in 10hrs), if I haven't been told otherwise.

Reply:
From a strictly compatibility perspective, we're probably good to just target `netstandard2.0`. Although it appears that the `netstandard2.1` target was originally added to support taking advantage of the nullability annotations that come built-in with `netstandard2.1`, I think the fact that we also target `net8.0` now means we have good coverage here.

A quick peruse of the code doesn't reveal that we're using any APIs that are only available in `netstandard2.1` and not `netstandard2.0` so I think we can remove it.

Reply:
Thanks @captainsafia. I'll try installing the package as a dependency of both a netstandard2.0 and netstandard2.1 project tomorrow, to see if there are any usability differences.

(I'd be happy for us to remove it in a separate PR if that makes anything easier, btw. We could get this one merged first, if that's useful.)

Reply:
Great. As we may remove netstandard2.1 anyway, I'll approve this and merge without you adding the condition back in, @thompson-tomo :)